--- a/internal/frontend/bridge-gui/bridge-gui/qml/Notifications/Notifications.qml
+++ b/internal/frontend/bridge-gui/bridge-gui/qml/Notifications/Notifications.qml
@@ -628,7 +628,7 @@ QtObject {
         brief: title
         description: Backend.goos === "darwin" ?
             qsTr("Bridge is not able to access your keychain. Please make sure your keychain is not locked and restart the application.") :
-            qsTr("Bridge is not able to detect a supported password manager (pass or secret-service). Please install and setup a supported password manager and restart the application.")
+            qsTr("Bridge is not able to detect a supported password manager (pass or secret-service). Please connect the password-manager-service with the following command: `sudo snap connect protonmail-bridge:password-manager-service`.")
         group: Notifications.Group.Dialogs | Notifications.Group.Configuration
         icon: "./icons/ic-exclamation-circle-filled.svg"
         linkText: qsTr("Learn more about keychain issues")
